onpremise/how-rd-works/data-source-results-format.md

```diff
@@ -1,5 +1,13 @@
 # **Data Source Results Format**
 
+Valid types of columns returned in results:
+TYPE_INTEGER = 'integer'
+TYPE_FLOAT = 'float'
+TYPE_BOOLEAN = 'boolean'
+TYPE_STRING = 'string'
+TYPE_DATETIME = 'datetime'
+TYPE_DATE = 'date'
```

Comment:
No, it has no meaning in this format (my bad - I just copy pasted the original list directly from the code).